source/helpers/bisect.tsx

```diff
@@ -77,6 +77,7 @@ async function onChoiceButtonClick({currentTarget}: React.MouseEvent<HTMLButtonE
 			bisectedFeatures.slice(getMiddleStep(bisectedFeatures))
 		);
 
+		currentTarget.parentElement!.replaceWith(<div className="btn btn-disabled">Reloading…</div>);
```

Comment:
`btn-disabled` doesn't seem to work. Does anyone know the correct class without having to use `<button>`?

Reply:
`aria-disabled="true"` seems to work